src/lancie-admin-layout/lancie-admin-layout.html

```diff
@@ -78,6 +78,10 @@
         border-radius: 5px;
         padding: 5px;
       }
+
+      .web {
+        border-top: 1px solid #e8e8e8;
```

Comment:
It's literally in line 72...

Reply:
Aah woops this was due to refactoring. Missed that!